homeassistant/components/velbus/cover.py

```diff
@@ -83,16 +84,28 @@ def current_cover_position(self) -> int | None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        await self._channel.open()
+        try:
```

Comment:
Let's fix a wrapper as @thecode propose to make the code a bit more dry.
Now it repeats a lot with the same statement